lfucg-exactions/client/js/components/AccountLedgerExisting.js

```diff
@@ -70,25 +70,27 @@ class AccountLedgerExisting extends React.Component {
                                     </h3>
                                 </div>
                             </div>
-                            <div className={accountLedger.is_approved ? 'row link-row' : 'row link-row-approval-pending'}>
-                                <div className="col-xs-12 col-sm-5 col-md-3 col-sm-offset-7 col-md-offset-9">
-                                    <div className="col-xs-5">
-                                        {currentUser && currentUser.permissions && currentUser.permissions.accountledger &&
-                                            <Link to={`account-ledger/form/${accountLedger.id}`} aria-label="Edit">
-                                                <i className="fa fa-pencil-square link-icon col-xs-4" aria-hidden="true" />
+                            <div className="row link-row">
+                                <div className={accountLedger.is_approved ? 'row link-row' : 'row link-row-approval-pending'}>
+                                    <div className="col-xs-12 col-sm-5 col-md-3 col-sm-offset-7 col-md-offset-9">
+                                        <div className="col-xs-5">
+                                            {currentUser && currentUser.permissions && currentUser.permissions.accountledger &&
+                                                <Link to={`account-ledger/form/${accountLedger.id}`} aria-label="Edit">
+                                                    <i className="fa fa-pencil-square link-icon col-xs-4" aria-hidden="true" />
+                                                    <div className="col-xs-7 link-label">
+                                                        Edit
+                                                    </div>
+                                                </Link>
+                                            }
+                                        </div>
+                                        <div className="col-xs-5 ">
+                                            <Link to={`account-ledger/summary/${accountLedger.id}`} aria-label="Summary">
```

Comment:
if we want to merge in the Credit Transfer changes this would need to say`<Link to={credit-transfer/summary/${accountLedger.id}}`